nifi-registry-core/nifi-registry-web-api/src/main/java/org/apache/nifi/registry/web/api/ExtensionResource.java

```diff
@@ -93,15 +94,25 @@ public ExtensionResource(final RegistryService registryService,
             responseContainer = "List"
     )
     @ApiResponses({ @ApiResponse(code = 401, message = HttpStatusMessages.MESSAGE_401) })
-    public Response getExtensionBundles() {
+    public Response getExtensionBundles(
+            @QueryParam("groupId")
+            @ApiParam("Optional groupId to filter results. The value may be an exact match, or a trailing wildcard, " +
+                    "such as 'com.%' to select all bundles where the groupId starts with 'com.'.")
+                final String groupId,
+            @QueryParam("artifactId")
+            @ApiParam("Optional artifactId to filter results. The value may be an exact match, or a trailing wildcard, " +
+                    "such as 'nifi-%' to select all bundles where the artifactId starts with 'nifi-'.")
+                final String artifactId) {
```

Comment:
These parameters are nice to have, and work as expected for me. One thing I noticed is that the documentation says _"... a trailing wildcard ..."_, which indicates wildcards can only be used at the end of the value. However in testing, I was able to use them anywhere in the value (at the beginning or middle of the value) and it worked (with the expected results as well).

So maybe just update the documentation to remove the work "trailing"?